src/apps/content-editor/src/app/views/ItemEdit/components/BlockTabs.tsx

```diff
@@ -0,0 +1,270 @@
+import {
+  Button,
+  Tabs,
+  Tab,
+  Box,
+  Typography,
+  List,
+  ListItemButton,
+  TextField,
+  InputAdornment,
+} from "@mui/material";
+import { useRef, useState } from "react";
+import { ThemeProvider } from "@mui/material/styles";
+import { theme } from "@zesty-io/material";
+import { VerticalSplitRounded, InfoRounded } from "@mui/icons-material";
+import { Actions } from "../Content/Actions";
+import { customTheme } from "../../../ContentEditor";
+import { useHistory, useParams } from "react-router";
+import { useGetContentModelItemsQuery } from "../../../../../../../shell/services/instance";
+import { ContentItem } from "../../../../../../../shell/services/types";
+import moment from "moment-timezone";
+import { useGetUsersQuery } from "../../../../../../../shell/services/accounts";
+import { AddRounded, SearchRounded } from "@mui/icons-material";
+import noSearchResults from "../../../../../../../../public/images/noSearchResults.svg";
+import blockPlaceholder from "../../../../../../../../public/images/blockPlaceholder.png";
+export const BlockTabs = (props: any) => {
+  const [value, setValue] = useState(0);
```

Comment:
This is hard to understand immediately. "Value" is to generic. What is the context of this value? 

This jumped out to me when it is equated later. `value == 0`. Very hard to know what that means beyond it's literal evaluation.